2.4/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -134,39 +134,41 @@ function replset_addr() {
 }
 
 # mongo_remove removes the current MongoDB from the cluster
+# optionally takes identifier of replset to connect and address of server to remove
```

Comment:
This says "removes the current MongoDB from the cluster". That is inaccurate.
If we are to make the function generic, let's revisit the wording here please.